src/hooks/use-action-dispatcher.js

```diff
@@ -1,23 +0,0 @@
-import * as React from 'react';
```

Comment:
(This file's behavior was inlined directly into `use-deck-state` for better type-safety)